Jenkinsfile

```diff
@@ -263,20 +263,29 @@ pipeline {
                                       context: "build" + "/" + env.STAGE_NAME,
                                       status: "PENDING"
                         checkoutScm withSubmodules: true
-                        sh label: env.STAGE_NAME,
-                           script: '''rm -rf artifacts/leap15/
-                              mkdir -p artifacts/leap15/
-                              if git show -s --format=%B | grep "^Skip-build: true"; then
-                                  exit 0
-                              fi
-                              make CHROOT_NAME="opensuse-leap-15.1-x86_64" -C utils/rpms chrootbuild'''
+                        catchError(stageResult: 'UNSTABLE', buildResult: 'SUCCESS') {
+                            sh label: env.STAGE_NAME,
+                               script: '''rm -rf artifacts/leap15/
+                                  mkdir -p artifacts/leap15/
+                                  if git show -s --format=%B | grep "^Skip-build: true"; then
+                                      exit 0
+                                  fi
+                                  if ! make CHROOT_NAME="opensuse-leap-15.1-x86_64" -C utils/rpms chrootbuild; then
+                                      cp /etc/mock/opensuse-leap-15.1-x86_64.cfg artifacts/leap15/;
+                                      exit 1
+                                  fi'''
+                        }
                     }
                     post {
                         success {
                             sh label: "Build Log",
-                               script: '''(cd /var/lib/mock/opensuse-leap-15.1-x86_64/result/ &&
+                               script: '''mockroot=/var/lib/mock/opensuse-leap-15.1-x86_64
+                                          (cd $mockroot/result/ &&
                                            cp -r . $OLDPWD/artifacts/leap15/)
-                                          createrepo artifacts/leap15/'''
+                                          createrepo artifacts/leap15/
+                                          rpm --qf %{version}-%{release}.%{arch} -qp artifacts/centos7/daos-server-*.x86_64.rpm > leap15-rpm-version
```

Comment:
Not sure that the RPM version stash is needed for CaRT but I wouldn't be against including it just to keep the CaRT and DAOS `Jenkinsfile`s in sync.  But if doing that, should get the cart RPM version:

```suggestion
                                          rpm --qf %{version}-%{release}.%{arch} -qp artifacts/centos7/cart-*.x86_64.rpm > leap15-rpm-version
```